packages/slider/src/presenters/input.scss

```diff
@@ -0,0 +1,80 @@
+@import "~@hig/styles/mixins/colors.scss";
+@import "~@hig/styles/mixins/slider.scss";
+
+.hig__slider__input {
+  -webkit-appearance: none;
+  width: 100%; /* Specific width is required for Firefox. */
+  background-color: transparent; /* Otherwise white in Chrome */
+  box-sizing: content-box;
+  height: 60px;
+  margin: 0;
+  position: relative;
+
+  &:hover {
+    @include thumb {
+      background-color: color(hig-turquoise-50);
+    }
+  }
+
+  &:focus {
+    outline: none;
+
+    @include thumb {
+      background-color: color(hig-blue-50);
+    }
+  }
+
+  &:active {
+    @include thumb {
+      background-color: color(hig-blue-60);
+    }
+  }
+
+  &[disabled] {
+    @include thumb {
+      background-color: color(hig-white);
+      border: 2px dotted color(hig-cool-gray-20);
+      outline: 2px solid color(hig-white);
+    }
+
+    @include track {
+      background-color: color(hig-cool-gray-10);
+    }
+  }
+
+  @include thumb {
+    box-sizing: content-box;
+    height: 16px;
+    width: 16px;
+    background: color(hig-cool-gray-20);
+    border-radius: 50%;
+    cursor: pointer;
+    position: relative;
+    top: 50%;
+    border: 2px solid color(hig-white);
+    -webkit-appearance: none;
+  }
+
+  &::-webkit-slider-thumb {
+    transform: translateY(-50%);
```

Comment:
I noticed that this rule applies to both Chrome and Edge. Edge's implementation does not require this transform, so the thumb-knob renders above the track.


![virtualbox_virtual_01_06_2018_15_12_01](https://user-images.githubusercontent.com/1744567/40865738-bdb9f300-65ae-11e8-9587-5b9b36cf9851.png)


Leaving this to fix later since it matches the current implementation.